sw/device/silicon_creator/rom_ext/sival/binaries/BUILD

```diff
@@ -53,9 +53,25 @@ filegroup(
     srcs = glob(["*.signed.bin"]),
 )
 
+filegroup(
+  name = "signed_spx_binaries",
+  srcs = ["//sw/device/silicon_creator/rom_ext:rom_ext_dice_x509_slot_a"],
+  output_group = "fpga_cw340_signed_bin",
+  testonly = True,
+)
+
 signature_test(
     name = "sival_signature_test",
+    spx_domain = "", # "Pure", "PrehashedSha256", "PrehashedSha256Reversed"
     srcs = [
         ":signed_binaries",
     ],
 )
+
+signature_test(
+    name = "sival_signature_spx_test",
+    spx_domain = "PrehashedSha256", # "Pure", "PrehashedSha256", "PrehashedSha256Reversed"
+    srcs = [
+        ":signed_spx_binaries",
+    ],
+)
```

Comment:
We should move this test somewhere else: This isn't testing anything related to the sival images, but rather its testing testing/development ROM_EXT in the codebase.  I'd suggest that the filegroup and this instantiation of the signature_test rule belong in `//sw/device/silicon_creator/rom_ext/BUILD`.